src/connectors/connector_spanner.workflows.json

```diff
@@ -40,7 +40,7 @@
     "create_spanner_session": {
       "call": "googleapis.spanner.v1.projects.instances.databases.sessions.create",
       "args": {
-        "database": "projects/placeholder/instances/placeholder-instance-name/databases/placeholder-database-name"
+        "database": "projects/PROJECT_ID/instances/SPANNER_INSTANCE/databases/SPANNER_DATABASE"
```

Comment:
if we are using ${sys.get_env("GOOGLE_CLOUD_PROJECT_ID")} elsewhere, shoudl we use it here too?

Reply:
Yes, done